pallets/stake/src/lib.rs

```diff
@@ -1080,9 +1150,8 @@ impl<T: Config> Module<T> {
 
 /// Add reward points to block authors:
 /// * 20 points to the block producer for producing a block in the chain
-impl<T> author_inherent::EventHandler<T::AccountId> for Module<T>
-where
-	T: Config + author_inherent::Config,
+impl<T: Config + author_inherent::Config> author_inherent::EventHandler<T::AccountId>
```

Comment:
```suggestion
impl<T: Config> author_inherent::EventHandler<T::AccountId>
```

I don't think that bound is necessary here. If that pallet isn't in the runtime, this trait impl is still valid.